testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -396,6 +408,350 @@ public void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingWithTrueAndFalseDataProvider")
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testSupportedNonPartitionedColumnMappingWrites(String mode, boolean statsAsJsonEnabled)
```

Comment:
It would be nice to have test with INSERT on Trino and SHOW STATISTICS to ensure that we write statistics using correct column names to transaction log.

Reply:
This test case already has both `INSERT` & `SHOW STATS` in Trino. I'm not sure about the suggestion here.

Reply:
My mistake.